CODEOWNERS

```diff
@@ -0,0 +1,50 @@
+# Root
+/ @benolayinka @laurensslats
+/CODEOWNERS @benolayinka
+/LICENSE @benolayinka
+
+# Build pipeline and tooling
+/Makefile @benolayinka @htdvisser @johanstokking
+/.github/workflows @benolayinka @htdvisser @johanstokking
+
+# Hugo
+/doc/themes @htdvisser @benolayinka
+/doc/layouts @htdvisser @benolayinka
```

Comment:
This should probably be a frontend/UX person instead; @kschiffer ?

@htdvisser did a theme as a Fab Friday or spare time project, but let's not make him responsible for this

Reply:
Thanks. This indeed shouldn't be me.